public/index.html

```diff
@@ -106,13 +106,68 @@ <h2 class="inline-block text-white text-3xl lg:text-4xl font-medium bg-brand-sky
   </div>
 
   <!-- community -->
-  <section class="bg-black">
+  <section class="bg-waves-bg aspect-video w-full bg-repeat bg-center bg-cover p-200">
     <div class="max-w-4xl mx-auto py-16 md:py-24 lg:py-36 px-5">
       <h3 class="text-4xl lg:text-5xl text-center text-white">
         Warsaw-based <span class="text-brand-green">community around zero-knowledge proofs</span>. We run events with lectures and workshops for researchers, hackers, and builders.
       </h3>
     </div>
 
+<!-- Speakers -->
+<div class="max-w-7xl mx-auto ">
+  <div id="speaker" class="pt-12 xl:pt-20 px-5 lg:px-0">
+    <div class="flex items-center flex-col justify-between pb-8">
+      <h3 class="text-4xl lg:text-5xl text-center text-white mb-10">Speakers</h5>
+      <a class="inline-block sm:w-56 text-white text-center bg-[#002F4F]/80 rounded-full lg:hover:bg-[#002F4F] transition-all ease-linear py-4 px-8"
+        href="#">Become
```

Comment:
```suggestion
        href="https://scrollzkp.typeform.com/zkWarsawSpeaker" target="_blank">Become
```

Reply:
and add some padding before PFPs

Reply:
or move to below all the speaker PFPs, like with sponsors below? and add 'Ticket' button, like sponsors below?